mmengine/model/weight_init.py

```diff
@@ -481,7 +481,7 @@ def __call__(self, module):
         from mmengine.runner.checkpoint import (_load_checkpoint_with_prefix,
                                                 load_checkpoint,
                                                 load_state_dict)
-        logger = MMLogger.get_instance('mmengine')
+        logger = MMLogger.get_current_instance()
```

Comment:
We can directly call print_log(xxx, logger='current') here, and do not need to get the current logger here.